toolkit/scripts/toolchain/container/Dockerfile

```diff
@@ -104,11 +103,10 @@ COPY [ "./toolchain_initial_chroot_setup.sh", \
        "./toolchain-jdk8-build.sh", \
        "./Awt_build_headless_only.patch", \
        "./check-system-ca-certs.patch", \
-       "./rpm-define-RPM-LD-FLAGS.patch", \
```

Comment:
Can we keep this patch both in the toolchain build and the RPM build of RPM? This patch has not been upstreamed, but we do have a fair number of spec files that expect its presence (due to its 10+ year tenure in the Fedora build of RPM).